src/lib.rs

```diff
@@ -1463,6 +1463,27 @@ impl<K, V, S> Default for OrderMap<K, V, S>
     }
 }
 
+impl<K, V, S> PartialEq for OrderMap<K, V, S>
+    where K: Eq + Hash,
```

Comment:
Hm we can wait a bit with K and merge the Equivalent PR. It's a generalization of the Borrow mechanism https://github.com/bluss/ordermap/pull/10